sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -89,20 +100,25 @@ export class KeyVaultBackupClient {
     readonly vaultUrl: string;
 }
 
+// Warning: (ae-forgotten-export) The symbol "DataAction" needs to be exported by the entry point index.d.ts
```

Comment:
Make sure to have no instances of `Warning` nor `undocumented` in your `.api.md` files 🌞

Reply:
Yup I purposely left that in until I get guidance for https://github.com/Azure/azure-sdk-for-js/pull/13470#discussion_r566949726

I'll fix that now that I know what the right thing to do is. Thanks!